### PR TITLE
Add a note about requirements.txt for new-to-python folks

### PR DESCRIPTION
**Pull request by fritz-astronomer**

### CODE REVIEW

**Review by paolaperaza — APPROVED** [review]
Made a quick edit here to make this intro a little more organized, but let me know what you think @fritz-astronomer. I'll let you finalize this with @jwitz and merge!

Albeit basic, I do think this is useful 😊

**Comment by fritz-astronomer** [comment]
👍

File: astro/develop-project.md [hunk 1]
```diff
@@ -214,6 +217,20 @@ Airflow providers are Python packages that contain all relevant Airflow modules
...
+Use the `include` folder to store additional utilities required by your DAGs. For example, templated SQL scripts and custom operators.
+
+In most cases, Astronomer recommends moving the `include` folder into the `dags` directory so that your DAGs can access your utility files.
+
+:::tip
+
+When you use the `astro deploy -—dags` command to deploy to Astro, the `include` folder in the Astro project directory is not deployed with your DAGs unless it is in the `dags` directory. Instead, it is built into the Docker image with your other project files and requires running `astro deploy`. See [Deploy DAGs only](deploy-code.md#deploy-dags-only).
+
+:::
+
```
Comment (jwitz) [inline astro/develop-project.md:233]:
```suggestion
```
This was addressed in another PR that merged recently. See https://docs.astronomer.io/astro/develop-project#add-utility-files

File: astro/develop-project.md [hunk 2]
```diff
@@ -191,9 +191,12 @@ The `airflow_settings.yaml` file includes a template with the default values for
 
 ## Add Python, OS-level packages, and Airflow providers
 
-Most DAGs need a Python package or OS-level package to run. If you’re using Airflow for a data science project, for example, you might need to install popular data science libraries such as [pandas](https://pandas.pydata.org/) or [NumPy (`numpy`)](https://numpy.org/). Adding a Python package to your Astro project is equivalent to running `pip install`.
+Most Airflow DAGs need an OS-level or Python package to run. For Python packages, there are two primary kinds that you might have to add to your Astro project:
 
-Airflow providers are Python packages that contain all relevant Airflow modules for a third-party service. For example, `apache-airflow-providers-amazon` includes all hooks, operators, and integrations you need to access services on Amazon Web Services (AWS) with Airflow. See [Provider packages](https://airflow.apache.org/docs/apache-airflow-providers/).
+- Python libraries. If you’re using Airflow for a data science project, for example, you might use a popular data science library such as [pandas](https://pandas.pydata.org/) or [NumPy (`numpy`)](https://numpy.org/).
```
Comment (jwitz) [inline astro/develop-project.md:196]:
```suggestion
- Python libraries. If you’re using Airflow for a data science project, for example, you might use a data science library such as [pandas](https://pandas.pydata.org/) or [NumPy (`numpy`)](https://numpy.org/).
```

File: astro/develop-project.md [hunk 3]
```diff
@@ -191,9 +191,12 @@ The `airflow_settings.yaml` file includes a template with the default values for
...
+- Airflow providers. Airflow providers are Python packages that contain all relevant Airflow modules for a third-party service. For example, `apache-airflow-providers-amazon` includes the hooks, operators, and integrations you need to access services on Amazon Web Services (AWS) with Airflow. See [Provider packages](https://airflow.apache.org/docs/apache-airflow-providers/).
+
+Adding the name of a package to the `packages.txt` or `requirements.txt` files of your Astro project installs the package to your Airflow environment or Deployment. The `requirements.txt` file uses pip, a package manager for Python packages and modules.
```
Comment (jwitz) [inline astro/develop-project.md:199]:
```suggestion
Adding the name of a package to the `packages.txt` or `requirements.txt` files of your Astro project installs the package to your Airflow environment or Deployment. Python packages are installed from your `requirements.txt` file using [pip](https://pypi.org/project/pip/).
```

**Pull request merged**